include/deal.II/lac/trilinos_tpetra_vector.h

```diff
@@ -48,6 +48,13 @@ namespace LinearAlgebra
   class ReadWriteVector;
 #  endif
 
+  /**
+   * A namespace for classes that provide wrappers for Trilinos' Tpetra vectors.
+   *
+   * This namespace provides wrappers for the Epetra_FEVector class from the
```

Comment:
Just checking: Is 
> the Epetra_FEVector class

a copy-paste error? I'll also check the link and source files, but maybe you'll beat me to the answer in a reply.

Reply:
Yeah, I think that this should be `Tpetra::Vector`.

Reply:
This was a copy-and-past error. I have changed it. Your hint with the doxygen configuration file also worked!

Reply:
Great, thanks a lot for fixing this!